packages/pipeline-editor/src/pipelineProperties.ts

```diff
@@ -36,6 +42,19 @@ const pipelineProperties = {
         parameter_ref: 'runtime',
         label: { default: 'Pipeline Runtime' }
       },
+      {
+        control: 'custom',
+        custom_control_id: 'StringControl',
+        parameter_ref: 'schedule_interval',
+        label: {
+          default:
```

Comment:
Could you work with @marthacryan or @ajbozarth and see if they could help get a better user experience.

Reply:
@marthacryan @ajbozarth could please take a look and see how to improve the UX

Reply:
sorry I didn't comment earlier, I actually though this through last week. I agree with Alan's comment below about leveraging the airflow presets. If we do that then this property can be an enum using the preset values and then the backend can interpret those values into the correct string for the runtimes that dont support the notation